src/app/app.component.ts

```diff
@@ -1,10 +1,9 @@
-import { Component } from '@angular/core';
+import { Component, ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
this is unnnesesary. You don't need to override nested styles. Define styles in a place where you need them. In case you need some global styles, you can define them in `main.scss`

Reply:
done